### PR TITLE
Add alt text to images that don't already have it

### LINKED ISSUE #1104: Add `alt` attributes to all images
Related: #1103

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #1104.

Staged at URLs under https://kw-staging-dartlang-2.firebaseapp.com.

### CODE REVIEW

File: src/_guides/language/language-tour.md [hunk 1]
```diff
@@ -758,7 +758,7 @@ units. Use the `runes` property to get the runes of a string.
 
 The following example illustrates the relationship between runes,
 16-bit code units, and 32-bit code points.
-Click the run button ( {% asset red-run.png %} )
+Click the run button {% asset red-run.png alt="" %}
```
Comment (chalin) [inline src/_guides/language/language-tour.md]: Maybe `alt="(>)"`, here and elsewhere?

Comment (chalin) [reply src/_guides/language/language-tour.md]: (Btw, my plan was to eventually replace these images with the appropriate Material Icon.)

Comment (kwalrath) [reply src/_guides/language/language-tour.md]: The image doesn't add anything for non-sighted users, so per the accessibility guidelines I've found (e.g. https://webaim.org/techniques/alttext/), I set it to an empty string.

Comment (kwalrath) [reply src/_guides/language/language-tour.md]: (These are screenshots, so I'm not sure the Material icon is appropriate here.)

Comment (chalin) [reply src/_guides/language/language-tour.md]: We don't actually want a screenshot here, we want to show a button (which we can color red if that is deemed necessary):

    Click the run button <i class="material-icons">play_circle_filled</i>.

As you can see from this DartPad screenshot:

> screen shot 2018-08-31 at 05 57 41

The DartPad run button is essentially a standard Material play button, so it seems appropriate IMHO:

> screen shot 2018-08-31 at 06 00 04

Let me know if you agree. If so, I'll create a followup issue.

Comment (kwalrath) [reply src/_guides/language/language-tour.md]: Sure, that sounds good to me. I didn't realize the DartPad button conformed to Material Design. :)

Comment (chalin) [reply src/_guides/language/language-tour.md]: Done: #1118

**Pull request merged**